lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1665,11 +1666,18 @@ ParseResult FIRStmtParser::parsePostFixFieldId(Value &result) {
   StringRef fieldName;
   if (parseFieldId(fieldName, "expected field name"))
     return failure();
-
+  auto indexV =
+      result.getType().cast<FIRRTLType>().cast<BundleType>().getElementIndex(
+          fieldName);
```

Comment:
Double cast shouldn't be necessary?
```suggestion
      result.getType().cast<BundleType>().getElementIndex(fieldName);
```